host/storage/storage_test.go

```diff
@@ -321,7 +327,7 @@ func TestRemoveCorrupt(t *testing.T) {
 
 	// initialize the storage manager
 	am := alerts.NewManager()
-	vm, err := storage.NewVolumeManager(db, am, cm, log.Named("volumes"))
+	vm, err := storage.NewVolumeManager(db, am, cm, log.Named("volumes"), DefaultSettings.SectorCacheSize)
```

Comment:
I wasn't setting the value to 0 because of this this FAIL: TestRemoveCorrupt (0.07s)
    storage_test.go:329: failed to initialize cache: must provide a positive size
FAIL